partiql-planner/src/main/kotlin/org/partiql/planner/internal/Env.kt

```diff
@@ -358,6 +360,11 @@ internal class Env(
                     matches.add(match)
                 }
             }
+            // TODO: Verify if this is the correct behavior
+            if (rootType is AnyType) {
+                val match = ResolvedVar.Local(StaticType.ANY, ordinal, rootType, listOf(pathPrefix) + path.steps)
+                matches.add(match)
```

Comment:
To temporarily unblock the path access on any type. This behavior is probably incorrect.

Reply:
Interesting. I'm not sure we can resolve a variable if the type is ANY/UNKNOWN. I feel this should just be a path expression on the root

Reply:
We need a refactor on PartiQLValueType before doing this. 

Consider:

```
tbl : <<{'a' : 1, 'b': 2}>>
```

We now infer the `tbl` having `PartiQLValueType` of Bag, which is then converted to StaticType `BAG(ANY)`. 

So if we do `SELECT tbl.a FROM tbl`. Without this, the inferencer would fail.

Reply:
I think this would break the defeasible reasoning paper (PartiQL Variable Resolution) written by @RCHowell .

If R is NOT known and S is KNOWN, then we must choose S. In this scenario, we wouldn't choose either due to ambiguity. I believe the recently merged PR #1344 solves this.

Reply:
Pick the #1344. Removed the temporary work around.